### PR TITLE
Support aws china and aws us gov partition

### PR DESCRIPTION
**Pull request by ri0day**
aws has partition concept
https://docs.aws.amazon.com/whitepapers/latest/aws-fault-isolation-boundaries/partitions.html
different partition use different saml authenticate endpoint and sts endpoint

### CODE REVIEW

**Comment by Abukodonosor** [comment]
Will check the code and get back to you!
Thanks :+1: 

If I have some questions, are you will to participate? @ri0day

**Comment by ri0day** [comment]
> Will check the code and get back to you! Thanks 👍
> 
> If I have some questions, are you will to participate? @ri0day

Sure

File: background/script.js [hunk 1]
```diff
@@ -343,15 +364,29 @@ async function assumeRole(
...
+        accessKeyId: AccessKeyId,
+        secretAccessKey: SecretAccessKey,
+        sessionToken: SessionToken,
+      },
+    };
+  }
+
   // AWS SDK is a module exorted from a webpack packaged lib
   // See 'library.name' in webpack.config.js
   const client = new webpacksts.AWSSTSClient(clientconfig);
```
Comment (Abukodonosor) [inline background/script.js:392]: We should make one adjustment.

```
ReferenceError: client config is not defined
    at assumeRole (script.js:392:46)

```

So it means that the client config is block scoped and his reference not coming to the `const client = new webpacksts.AWSSTSClient(clientconfig);`

Pls use var or instead make the default the aws credentials and if the AWSPartition is "aws-cn" then rewrite it.

**Comment by Abukodonosor** [comment]
Also, one more thing left.

If you have time, will be nice to tackle it, if not its ok.

(The extensions support multi-input, but work only for one)
It should be a simple fix.[ but its not in this scope ]

You decide.

**Pull request merged**